MdePkg/Library/BaseFdtLib/BaseFdtLib.inf

```diff
@@ -57,6 +57,9 @@
   BaseMemoryLib
 
 [BuildOptions]
-  MSFT:*_*_IA32_CC_FLAGS = /wd4146 /wd4245
-  MSFT:*_*_X64_CC_FLAGS  = /wd4146 /wd4244 /wd4245 /wd4267
+# warning C4706: assignment within conditional expression

```

Comment:
Please consider commit message for more info like example below:
Disable some compiling warnings caused by submodule code.
Those can be removed later once issues fixed by submodule owner.